tools/run_tests/run_xds_tests.py

```diff
@@ -56,17 +56,27 @@
     'secondary_locality_gets_requests_on_primary_failure',
     'traffic_splitting',
 ]
+# Valid test cases, but not in all. So the tests can only run manually, and
+# aren't enabled automatically for all languages. Move them into _TEST_CASES
```

Comment:
Nit: Pull out the TODO into a separate labeled comment, e.g.:

```
# Valid test cases, but not in all. So the tests can only run manually, and
# aren't enabled automatically for all languages. 
# TODO: Move them into _TEST_CASES when support is ready in all languages.
```

Reply:
Done